docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -0,0 +1,28 @@
+# Kubeadm control plane
```

Comment:
This doc currently seems to be more around just the upgrade process. I'm wondering if it might be better to title it 'Upgrading workload clusters' and have a short blurb that the use of KubeadmControlPlane and MachineDeployment is a requirement for supported upgrades.

Reply:
The intent is that this is a place to put tasks related to kubeadm control plane. Upgrades just happen to be the most interesting feature. There should probably be some sentences about scaling up and down, fixing a busted control plane node and other tasks that kubeadm control plane allows us to do